shared/src/arguments.rs

```diff
@@ -50,6 +50,17 @@ pub struct Arguments {
     /// List of token addresses to be ignored throughout service
     #[structopt(long, env = "UNSUPPORTED_TOKENS", use_delimiter = true)]
     pub unsupported_tokens: Vec<H160>,
+
+    /// Which Liquidity sources to be used by Price Estimator.
+    #[structopt(
```

Comment:
Here I'd say we should reuse AMM_SOURCES (maybe it can be renamed to BASELINE_SOURCES)